libs/headless-lfp/src/test/language-font-finder.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { act, renderHook } from '@testing-library/react';
+
+import useLanguageFontFinder from '../language-font-finder';
+
+describe('useLanguageFontFinder', () => {
+  it('should render successfully', async () => {
+    const { result } = renderHook(() => useLanguageFontFinder());
+
+    const {error,finding } = result.current
```

Comment:
Running `nx format:write` should fix this formatting issue and you can also setup VS Code to do it for you. You might consider running `nx format:check` in `lint.yml` before running the linter.

Reply:
Good call. Coming up!